packages/forms/src/styled/checkbox/StyledCheckInput.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+import { getColor, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { StyledRadioInput } from '../radio/StyledRadioInput';
+import { StyledCheckLabel } from './StyledCheckLabel';
+
+const COMPONENT_ID = 'forms.checkbox';
+
+const checkSvg = props => {
+  const color = props.theme.colors.background;
+
+  return `
+    <svg xmlns="http://www.w3.org/2000/svg" width="12" height="12">
+      <path
+        fill="none"
+        stroke="${color}"
+        stroke-linecap="round"
+        stroke-linejoin="round"
+        stroke-width="2"
+        d="M3 6l2 2 4-4"
+      />
+    </svg>
+  `;
+};
+
+const dashSvg = props => {
+  const color = props.theme.colors.background;
+
+  return `
+    <svg xmlns="http://www.w3.org/2000/svg" width="12" height="12">
+      <path
+        fill="none"
+        stroke="${color}"
```

Comment:
Just to confirm, the `currentColor` value wouldn't work here since we are apply this as a background-image?

Reply:
Correct, it would not work.